packages/vite/index.cjs

```diff
@@ -25,6 +25,10 @@ asyncFunctions.forEach((name) => {
 
 function warnCjsUsage() {
   if (process.env.VITE_CJS_IGNORE_WARNING) return
+  const isSilent = process.argv.some((arg) =>
+    /^(-l|--logLevel)=(['"]?)silent\2$/.test(arg),
```

Comment:
Maybe we should do similarly like this https://github.com/vitejs/vite/blob/ebfaa7e6019783f308db5e759a6a67abb7cc626c/packages/vite/bin/vite.js#L15-L17 so that it also handles `--logLevel silent`. And I think technically `--logLevel error` would also omit the warning.

The idea sounds nice though.

Reply:
Thanks for the quick review, updated.